public/src/pages/auth/Login/Login.tsx

```diff
@@ -152,10 +152,13 @@ export const Login: FC = () => {
           {oidcClient && (
             <div>
               <p>
-                <b>clientId:</b> {oidcClient.clientId}
+                <b>client_id:</b> {oidcClient.clientId}
               </p>
               <p>
-                <b>clientSecret:</b> {oidcClient.clientSecret}
+                <b>client_secret:</b> {oidcClient.clientSecret}
+              </p>
+              <p>
+                <b>oidc-config URL:</b> {oidcClient.metadataUrl}
```

Comment:
Don't use abbr here. Please keep the same wording
```suggestion
                <b>Openid-configuration URL:</b> {oidcClient.metadataUrl}
```